voting-device/voting-device.py

```diff
@@ -10,66 +10,81 @@
 config = configparser.ConfigParser()
 config.read('device_config.ini')
 firebase_api_key = config['TOKENS']['firebase_api_key']
-refresh_token = config['TOKENS']['refresh_token']
-
-first_votes, second_votes = 0,0
+auth_token = config['TOKENS']['auth_token']
+base_url = config['URL']['base_url']
+session = requests.Session()
+first_votes, second_votes = 0, 0
+cookies = None
 
 def main():
     global first_votes, second_votes
     login()
     me()
+    pollid = config['POLL']['pollid']
+    print('voting on poll id:', pollid)
     while True:
         char = input('insert 1 to vote first, 2 for vote second, s for send and r for reset:\n')
-        if char not in ['1','2','s','r']:
+        if char not in ['1', '2', 's', 'r']:
             print('illegal input')
             continue
         if char == '1':
-            first_votes +=1
+            first_votes += 1
         if char == '2':
```

Comment:
Her skal du vel bruke `elif`

Reply:
må ha det sånn for å printe ut current votes på alt unntatt send_votes